src/posix/client.cpp

```diff
@@ -170,41 +170,23 @@ static bool DoWrite(int aFile, const void *aBuffer, size_t aSize)
 
 static void SendBlockingCommand(int aArgc, char *aArgv[])
```

Comment:
nit: Is this method name still appropriate given that we are no longer waiting for a response?